### ISSUE TITLE
Code Table Request - new code table ctconcentration

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6797]
### Goal

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspecpart_attribute_type#dna_concentration / https://arctos.database.museum/info/ctDocumentation.cfm?table=ctspec_part_att_att#dna_concentration is terribly misfigured. DNA concentration needs appropriate units.

### Context

The current units table is nonsensical.

I don't see any appropriate existing units tables.

### Table

TBD

### Proposed Value

Suggest new table ``ctconcentration`` but perhaps ``ctratio`` or ???? would be more modular.

(The first data value in the table request will be for ``ng/μl``, which will be made in a new Issue after an appropriate table exists.)

### Helpful Actions

- [ ] Add the issue to the [Code Table Management Project](https://github.com/ArctosDB/arctos/projects/13#card-31628184).

- [ ] Please reach out to anyone who might be affected by this change. Leave a comment or add this to the Committee agenda if you believe more focused conversation is necessary.

@ArctosDB/arctos-code-table-administrators 

## Approval
_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [ ] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [ ] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [ ] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection
_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation
_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

- [ ] Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel.

- [ ] Add or revise the code table term/definition as described above. Ensure the URL of this Issue is included in the definition. URLs should be included *_as text_*, separated by spaced pipes. Do not include HTML in definitions.

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Jegelewicz** [comment]
I'm in favor of ctratio because it has broader application.

**Comment by dustymc** [comment]
Is it still a ratio if it requires units?

**Comment by Jegelewicz** [comment]
Yeah - but what about when we need miles per hour or something like that - wouldn't it go in this table? Maybe that's just getting too theoretical and also maybe just totally something else...

**Comment by dustymc** [comment]
Ah - yea, I'm not sure, we clearly have had too many tables (length units - and nothing else - in a few places, all spelled differently), but clearly there's some room for more-specialized tables ("random things used by THAT"). I sorta-maybe think we should do whatever's necessary to keep furlongs/fortnight away from DNA concentration, but I suppose I could also see that as nothing too hard to avoid (and misuse would be a useful quality indicator...).

Halfheared vote for 'concentration' but I can go either way.

(And The Official Arctos Mathlete says ratios can in fact have units.)

**Comment by Jegelewicz** [comment]
> ratios can in fact have units

I mean, at least the same units as the tow things being compared just so we know what the heck they were?

I'm good with ctconcentration (except it makes me feel like I should be concentrating...)

**Comment by dustymc** [comment]
I don't think the misconfigured attribute that's waiting on this to get fixed allows this to drag out. Is there some (effective) way to prioritize, or should I just do this, or should I dump some data and un-do the timebomb, or ????????? Suppose I'll do something or another soonish if I must. HELP!

**Comment by campmlc** [comment]
I'm good with just doing this as ctconcentration.  I checked a box.

**Comment by mkoo** [comment]
I meant to chime in a day ago that ctconcentration seems rational to me (see what i did there) - box checked

**Comment by dustymc** [comment]
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctconcentration
